database/sqlite/sqlite_aclk.c

```diff
@@ -455,52 +406,12 @@ static void timer_cb(uv_timer_t* handle)
 #endif
 }
 
-
-#ifdef ENABLE_ACLK
-void after_send_retention(uv_work_t *req, int status)
-{
-    struct aclk_database_worker_config *wc = req->data;
-    (void)status;
-    stop_retention_run();
-    wc->retention_running = 0;
-
-    struct aclk_database_cmd cmd;
-    memset(&cmd, 0, sizeof(cmd));
-    cmd.opcode = ACLK_DATABASE_DIM_DELETION;
-    if (aclk_database_enq_cmd_noblock(wc, &cmd))
-        info("Failed to queue a dimension deletion message");
-
-    cmd.opcode = ACLK_DATABASE_NODE_INFO;
```

Comment:
Hi Timo!

This was piggy-backed there to do a NODE INFO send every `ACLK_DATABASE_ROTATION_INTERVAL`. Not this PR's problem I think, as long as we do a follow up one (I can do it) to have the functionality again.

Reply:
It can go to the CLEANUP interval for now -- A follow up PR will need to clean the tables completely

Reply:
I added this